man/timedatectl.xml

```diff
@@ -146,6 +146,17 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>show</command></term>
+
+        <listitem><para>Show properties of the manager of
+        <citerefentry><refentrytitle>systemd-timedated.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+        By default, empty properties are suppressed. Use <option>--all</option> to show those too. To select specific
+        properties to show, use <option>--property=</option>. This command is intended to be used whenever
+        computer-parsable output is required. Use <option>status</option> if you are looking for formatted
+        human-readable output.</para></listitem>
```

Comment:
I don't like the first sentence — "manager" is our internal jargon. I think it'd be better to move the last sentence first, because it actually describes what this verb does:
```
<para>Show the same information as <option>status</option>, but in machine readable form. This command is intended to be used whenever computer-parsable output is required.<para>

<para>By default, empty properties are suppressed. Use <option>--all</option> to show those too. To select specific properties to show, use <option>--property=</option>.</para>
```